### ISSUE TITLE
ExceptionHandler cannot be registered to StreamObserverReadStream in manyToMany

### ISSUE THREAD

**Issue by giri-vsr** [vert-x3/vertx-grpc#141]
### Version
4.4.6

### Context

when `io.vertx.grpc.stub.ClientCalls.manyToMany` called if exception happens before returning the `StreamObserverReadStream ` exception is lost.There is no way to register exceptionHandler before the call.We can register exceptionHandler only after `StreamObserverReadStream` return from `manyToMany`.

```
  public static <I, O> ReadStream<O> manyToMany(ContextInternal ctx, Handler<WriteStream<I>> requestHandler, Function<StreamObserver<O>, StreamObserver<I>> delegate) {
    StreamObserverReadStream<O> response = new StreamObserverReadStream<>();
    StreamObserver<I> request = delegate.apply(response);
    requestHandler.handle(new GrpcWriteStream<>(request));
    return response;
  }
```

Exception happens in `StreamObserver<I> request = delegate.apply(response)` and `onError` is called and `exceptionHandler` is null. 

https://github.com/etcd-io/jetcd/issues/1261#issuecomment-1803304415

**Comment by tsegismont** [comment]
Fixed by e1848f2

**Comment by giri-vsr** [comment]
> > Fixed by [e1848f2](https://github.com/vert-x3/vertx-grpc/commit/e1848f2aab4fc9f01ea8ab9176a38645e6df32c2)
> 
> when can I expect 4.5.1 to be released?

@tsegismont will it be released soon?

**Comment by giri-vsr** [comment]
@vietj can I get some idea of when 4.5.1 will be released?
